### ISSUE TITLE
Accordion: invalid aria attribute

### ISSUE THREAD

**Issue by silvalaura** [cengage/react-magma#2483]
**Describe the bug**
During the Axe scanning we are getting the error in https://react-magma.cengage.com/version/4.13.0/api/accordion/ also find the attached screenshot for reference. Please have a look and share if anyone have any thought on it

([Slack](https://cengage.slack.com/archives/C8Y4FR63U/p1777909426591999))

**Screenshots**

<img width="1166" height="175" alt="Image" src="https://github.com/user-attachments/assets/77dc62b2-fc00-4396-8c46-d707544ba503" />

<img width="1572" height="755" alt="Image" src="https://github.com/user-attachments/assets/fb554b02-2014-48c1-8d78-d84615e423bf" />

<img width="934" height="722" alt="Image" src="https://github.com/user-attachments/assets/2223f2a7-26d4-4d7b-a7d0-c1ddad5c04ff" />

**Laura's Observations**
This does not come up on our storybook a11y tab in v4, but does in v5. The adopter who reported this is using React v18.3.1.

**Comment by nikitaorliak-cengage** [comment]
**Tested on:**
- V5: https://storybook-preview-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/accordion--default
- V4: https://storybook-preview-v4-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/accordion--default

Everything works as expected. No bugs found.
